src/app/rundown-view/components/on-air-details-panel/on-air-details-panel.component.html

```diff
@@ -1,23 +1,23 @@
 <sofie-spacer></sofie-spacer>
 
-<div class="sub-panel">
-    <span class="sub-panel__label">Part countdown</span>
-    <span class="sub-panel__value">00:04</span>
+<div class="subpanel">
```

Comment:
The class is called `on-air-details-panel`. It doesn't know that it is being used as a `subpanel`. That's a detail of where it's being used, not a detail the component itself should know about.

Reply:
The subpanel class is used for subpanels inside the `on-air-details-panel`.